contracts/contracts/vault/VaultAdmin.sol

```diff
@@ -163,6 +163,20 @@ contract VaultAdmin is VaultStorage {
         emit DripperChanged(_dripper);
     }
 
+    /**
+     * @notice Changes the async withdrawal claim period for OETH & superOETHb
+     * @param _delay Delay period (should be between 10 mins to 7 days).
+     *          Set to 0 to disable async withdrawals
+     */
+    function setWithdrawalClaimDelay(uint256 _delay) external onlyGovernor {
+        require(
+            _delay == 0 || (_delay >= 10 minutes && _delay <= 7 days),
```

Comment:
I think it makes sense to set the max delay to something that's greater than the max validator withdraw time. Perhaps 10 days? (At least one LST has a 15 day timed withdraw)

Reply:
Since this is just an upper bound, not the actual value that we would be using, I just set it to 15 days